packages/nextra/styles/steps.css

```diff
@@ -1,11 +1,16 @@
-.nextra-steps h3 {
-  counter-increment: var(--counter-id);
-
-  &:before {
-    @apply _absolute _size-[33px];
-    @apply _border-4 _border-[rgb(var(--nextra-bg))] _bg-gray-100 dark:_bg-neutral-800;
-    @apply _rounded-full _text-neutral-400 _text-base _font-normal _text-center _-indent-px;
-    @apply _mt-[3px] _ml-[-41px];
-    content: counter(var(--counter-id));
+.nextra-steps {
+  h2,
+  h3,
+  h4 {
+    counter-increment: var(--counter-id);
+    @apply _relative _mt-8 _border-none _text-xl _font-semibold;
+    &::before {
+      @apply _absolute _size-[33px];
+      @apply _border-4 _border-[rgb(var(--nextra-bg))] _bg-gray-100 dark:_bg-neutral-800;
+      @apply _rounded-full _text-neutral-400 _text-base _font-normal _text-center _-indent-px;
+      /* @apply _mt-[3px] _ml-[-41px]; */
+      @apply _top-[calc(50%-16.5px)] _left-[-41px];
```

Comment:
this brokes steps on ltr
<img width="918" alt="image" src="https://github.com/user-attachments/assets/f43182e3-c4a2-422e-b207-0438ce09936f">

Reply:
(I think it was already broken in v3, but I fixed in v4)